chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -828,6 +828,7 @@ sync.localDataWillBeCombined		= If you continue, local Zotero data will be combi
 sync.localGroupsWillBeRemoved1		= Local groups, including any with changed items, will also be removed from this computer.
 sync.avoidCombiningData				= To avoid combining data, revert to the '%S' account or use the Reset options in the Sync pane of the Zotero preferences.
 sync.unlinkWarning					= Are you sure you want to unlink this account?\n\n%S will no longer sync your data, but your data will remain locally.
+sync.warning.emptyLibrary			= Your are about to sync to an empty %1$S database. This might be due to a new installation %1$S or the location of your data directory changing.\n\nDo you want to continue?
 
```

Comment:
This actually shouldn't happen for a new Zotero installation, because a new installation will need to set the userID in the prefs before syncing, right? So this would only be a new database, either from a deleted previous database or a new data directory, with an existing Firefox or Zotero profile with an API key stored.

I also think we should provide a third button to open the data dir prefs.

sync.warning.emptyDatabase = You are about to sync the ‘%1$S’ account to an empty %2$S database. This could happen if you removed your previous %2$S database or if the location of your data directory changed.
sync.warning.existingDataElsewhere = If your %1$S data exists elsewhere on your computer, you should move it to your current data directory or change your data directory location to point to the existing data.

dataDir.changeDataDirectory = Change Data Directory…